src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/http_action/config/PerformanceAnalyzerOverridesClusterConfigAction.java

```diff
@@ -175,6 +175,12 @@ private boolean validateOverrides(final ConfigOverrides requestedOverrides) {
             isValid = Collections.disjoint(requestedOverrides.getEnable().getActions(), requestedOverrides.getDisable().getActions());
         }
 
```

Comment:
nit: Let's add a comment similar to `// Check if any remediation actions are in both enabled and disabled lists`